ducktape/tests/test.py

```diff
@@ -142,32 +142,42 @@ def __init__(self, **kwargs):
         :param cls
         :param function
         :param injected_args
+        :param service_registry
+        :param _cluster_size
         """
         self.session_context = kwargs.get("session_context", None)
         self.module = kwargs.get("module", None)
         self.cls = kwargs.get("cls", None)
         self.function = kwargs.get("function", None)
         self.injected_args = kwargs.get("injected_args", None)
         self.ignore = kwargs.get("ignore", False)
+        self._cluster_size = kwargs.get("_cluster_size", None)
 
         self.services = ServiceRegistry()
 
         # dict for toggling service log collection on/off
         self.log_collect = {}
 
     def __repr__(self):
-        return "<module=%s, cls=%s, function=%s, injected_args=%s>" % \
-               (self.module, self.cls_name, self.function_name, str(self.injected_args))
+        return "<module=%s, cls=%s, function=%s, injected_args=%s, cluster_size=%d>" % \
+               (self.module, self.cls_name, self.function_name, str(self.injected_args),
+                self.cluster_size)
 
     def copy(self, **kwargs):
-        """Construct a new TestContext object from another TestContext object
-        Note that this is not a true copy, since a fresh ServiceRegistry instance will be created.
-        """
+        """Construct a new TestContext object from another TestContext object"""
         ctx_copy = TestContext(**self.__dict__)
         ctx_copy.__dict__.update(**kwargs)
         return ctx_copy
 
     @property
+    def cluster_size(self):
```

Comment:
Can we just get rid of this and use the field directly by accounting for the value missing in the constructor?